tests/components/ecobee/test_number.py

```diff
@@ -106,17 +106,96 @@ async def test_set_compressor_protection_min_temp(hass: HomeAssistant) -> None:
 
     Ecobee runs in Fahrenheit; the test rig runs in Celsius. Conversions are necessary
     """
-    target_value = 0
-    with patch(
-        "homeassistant.components.ecobee.Ecobee.set_aux_cutover_threshold"
-    ) as mock_set_compressor_min_temp:
-        await setup_platform(hass, NUMBER_DOMAIN)
+    target_value = 10
+    await setup_platform(hass, NUMBER_DOMAIN)
 
+    with (
+        patch(
+            "homeassistant.components.ecobee.Ecobee.set_aux_cutover_threshold"
+        ) as mock_set_compressor_min_temp,
+        patch(
+            "homeassistant.components.ecobee.Ecobee.set_aux_maxtemp_threshold"
+        ) as mock_set_aux_max_temp_threshold,
+    ):
         await hass.services.async_call(
             NUMBER_DOMAIN,
             SERVICE_SET_VALUE,
             {ATTR_ENTITY_ID: COMPRESSOR_MIN_TEMP_ID, ATTR_VALUE: target_value},
             blocking=True,
         )
-        await hass.async_block_till_done()
-        mock_set_compressor_min_temp.assert_called_once_with(1, 32)
+
+        mock_set_compressor_min_temp.assert_called_once_with(1, 50)
+        mock_set_aux_max_temp_threshold.assert_called_once_with(1, 55)
+
+        target_value = 18
+        await hass.services.async_call(
+            NUMBER_DOMAIN,
+            SERVICE_SET_VALUE,
+            {ATTR_ENTITY_ID: COMPRESSOR_MIN_TEMP_ID, ATTR_VALUE: target_value},
+            blocking=True,
+        )
+
+        assert mock_set_compressor_min_temp.call_count == 2
+        calls = mock_set_compressor_min_temp.call_args_list
+        assert calls[1].args == (1, 64)
+
+        assert mock_set_aux_max_temp_threshold.call_count == 2
+        calls = mock_set_aux_max_temp_threshold.call_args_list
+        assert calls[1].args == (1, 69)
+
+
+AUX_MAX_OUTDOOR_TEMP_ID = "number.ecobee2_auxiliary_maximum_outdoor_temperature"
```

Comment:
Constants are normally defined at the top of the module.

Reply:
Fixed.